bedrock/firefox/views.py

```diff
@@ -636,12 +636,15 @@ def download_thanks(request):
     locale = l10n_utils.get_locale(request)
     variant = request.GET.get('v', None)
     show_newsletter = locale in ['en-US', 'en-GB', 'en-CA', 'en-ZA', 'es-ES', 'es-AR', 'es-CL', 'es-MX', 'pt-BR', 'fr', 'ru', 'id', 'de', 'pl']
-    pre_download_locales = ['en-US', 'en-GB', 'en-CA', 'en-ZA', 'de', 'fr']
+    pre_download_locales = ['fr']
 
     # ensure variant matches pre-defined value
-    if variant not in ['a']:  # place expected ?v= values in this list
+    if variant not in ['a', 'b', 'c', 'd']:  # place expected ?v= values in this list
         variant = None
 
+    if variant in ['b', 'c'] and locale == 'en-US':
+        show_newsletter = False
+
     if variant == 'a' and locale in pre_download_locales:
```

Comment:
Hm, is the pre-download experiment using variant a as well? Should this experiment should use something different. x,y,z, maybe?

Reply:
Seems like `a` is the control for quite a few experiments so maybe this isn't a problem.

Reply:
Yep, this test is en-US only so shouldn't interfere with the pre-download locales.